### DISCUSSION TITLE
Add some musical instruments (mainly concerning the bell)

### DISCUSSION THREAD

**Discussion by RaidhoSketch** [Roadhog360/Et-Futurum-Requiem#262] [Ideas]
The golden bell must absolutely be backported. Because surely, _something_ must be used for prayers in a shrine, something simple and sound-producing, and we are far too tired with block punching and wind chimes from Totemic. Actually, yeah, vanilla wind chimes sound awesome too - or any other simple passive or active music-producing item and/or block! Maybe a flute? It could absolutely be used as a mechanic for leading passive mobs without their respective food. Just, throwing ideas around.

**Comment by Roadhog360** [comment]
I don't understand the request. All of the note block notes you just asked for are already in the mod.

**Comment by RaidhoSketch** [comment]
Well, duh. Note blocks are the programmable ones, requiring quite a large setup to play any complex melody. Mainly, we asked for the bell block backport, which just plays a single sound, is aesthetically pleasing, and requires only a single block to be played.
